app/assets/stylesheets/application.scss

```diff
@@ -9,6 +9,7 @@ $covid-yellow: #fff500;
 @import 'govuk_publishing_components/govuk_frontend_support';
 @import 'govuk_publishing_components/component_support';
 @import 'govuk_publishing_components/components/accordion';
+@import 'govuk_publishing_components/components/action-link';
```

Comment:
I keep meaning to add a comment about this! It worked before because the action link (and styles) is imported by static (for the coronavirus header/banner). I decided not to include the action link styles here to avoid duplication.

It would be safer to include them here as well, in case we take the action link out of static at some point, however duplicating the CSS unnecessarily seems like a waste. But I'm happy with either decision.

Reply:
Ah, right. So I need to update static to get the new version of this component.

Reply:
Seems to be updated now, will re-check and if all good, I'm happy to drop this commit.

Reply:
Ah of course, yes, I forgot that until static is updated you don't get the latest styles. Yeah, all a bit confusing. Maybe change the commit to add that comment I think we need?